tools/tapenade/Dockerfile

```
@@ -5,18 +5,22 @@ WORKDIR /home/gradbench
 
 RUN apt-get update && apt-get install -y \
     bash \
-    gcc \
+    build-essential \
     python3
 
 # install Tapenade
 RUN wget https://tapenade.gitlabpages.inria.fr/tapenade/distrib/tapenade_3.16.tar
 RUN tar -xvzf tapenade_3.16.tar
 
-
 # ensure Tapenade is accessible
 RUN ln -s /home/gradbench/tapenade_3.16/bin/tapenade /usr/local/bin/tapenade
 
+# Make the cpp directory available in a somewhat odd location.
+COPY cpp /cpp/
+
 COPY tools/tapenade/ .
```

I realize now that I forgot to include DiffSharp SciLean, and Tapenade when I did this (and also I didn't document this principle anywhere), but in #80 I established the convention that, when we `COPY` files from this repo into a Docker image, we preserve their relative locations under `/home/gradbench`; could we do the same here?

```suggestion
COPY cpp /home/gradbench/cpp
COPY tools/tapenade /home/gradbench/tools/tapenade
```

---

Sure, I don't think that should be a problem.

---

Done.